script/setup

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -e
+
+brew bundle check || brew bundle
+
+# Prepare custom settings for Karabiner-Elements
+mkdir -p ~/.config/karabiner/
+ln -sf $PWD/karabiner/karabiner.json ~/.config/karabiner/karabiner.json
+
+# Prepare custom settings for Hammerspoon
+ln -sf $PWD/hammerspoon ~/.hammerspoon
+
+# Prepare custom settings for navigating between words in iTerm2
+cat $PWD/inputrc >> ~/.inputrc
```

Comment:
Pedantry: may want to check the contents aren't already present to avoid appending multiple times.